app/views/catalog/_show_with_viewer.html.erb

```diff
@@ -1,4 +1,7 @@
-<% if document.embeddable? %>
+<% if document.iiifable? %>
```

Comment:
Leaving this for @ggeisler ; what's the relative priority of IIIFable stuff we display in a DLME-specific mirador viewer vs oembeddable resources to display in a partner-provided viewer (e.g. sul-embed, or A/V, 3D, or PDF viewers)?